services/app-web/src/pages/Dashboard/Dashboard.test.tsx

```diff
@@ -66,18 +67,24 @@ describe('Dashboard', () => {
             name: 'Bob it user',
             email: 'bob@dmas.mn.gov',
         }
+        const draft = mockDraftSubmission2()
+        const submitted = mockSubmittedSubmission2()
+        const unlocked = mockUnlockedSubmission2()
+        
+        const submissions = [draft, submitted, unlocked]
 
-        const submissions = [mockDraft(), mockStateSubmission(), mockDraft()]
-        submissions[2].id = 'test-abc-122'
-        submissions[2].name = 'MN-MSHO-0002' // the names collide otherwise
-        // set middle row to latest updatedAt to test sorting (it should be sorted to top)
-        submissions[1].updatedAt = '2100-01-01T00:00:00.000Z'
+        // TODO - Figure this test mocking out
+
```

Comment:
Reminder for myself  - come back here and figure this out. This is the one failing app-web test because of the way things are mocked